### PR TITLE
Made changes to the Sanford Mysamples Page

### PR DESCRIPTION
**Pull request by ramavavi**
The issue address is : https://github.com/episphere/connect/issues/688#issue-1835562887

The changes made in the code are:

1. Added new location.
2. Removed hours from the two locations.

### CODE REVIEW

File: js/pages/samples.js [hunk 1]
```diff
@@ -521,7 +527,7 @@ const locations = [
 
 const renderLocations = (site) => {
     let template = '';
-
+console.log("site",site)
```
Comment (bransteitterbr) [inline js/pages/samples.js:530]: This should be removed

File: js/pages/samples.js [hunk 2]
```diff
@@ -314,19 +313,26 @@ const sanford = {
...
-        ]
+            'Sioux Falls: Sanford Imagenetics',
+            '1321 W. 22nd St.<br>Sioux Falls, SD 57104',
+            '',
+            'Free patient parking available is available in the lot near the front entrance--Door JJ. No parking validation needed.'
+        ],
+
+        [
+           'Fargo: South University Urgent Care/Orthopedic Walk-In Clinic',
+           '1720 University Drive S<br>Fargo, ND 58102<br> Enter Through Door #8',
```
Comment (JoeArmani) [inline js/pages/samples.js:332]: I think the space after the `<br>` and before 'Enter Through Door #8' needs to be removed.
`'1720 University Drive S<br>Fargo, ND 58102<br>Enter Through Door #8',`

File: js/pages/samples.js [hunk 3]
```diff
@@ -314,19 +313,26 @@ const sanford = {
...
-    howToDonate: 'The MyChart message or email we send you will contain a link to schedule an appointment. Simply click the link to schedule a time that is convenient for you to donate your samples.',
+    howToDonate: 'The MyChart message or email we send you will contain a link to schedule an appointment. Simply click the link to schedule a time that is convenient for you to donate your samples.Be sure you are scheduling an appointment in the city you are closest to.',
```
Comment (JoeArmani) [inline js/pages/samples.js:316]: need a space before 'Be sure you are...'

**Review by JoeArmani — CHANGES_REQUESTED** [review]
just a couple small items noted

File: js/pages/samples.js (hunk 3 above)
Comment (ramavavi) [reply js/pages/samples.js]: Added the space.

File: js/pages/samples.js (hunk 2 above)
Comment (JoeArmani) [reply js/pages/samples.js:332]: Agreed, but I'm referring to the space directly after the 2nd `<br>`. There is a difference between your code on line 332 and my code comment. Thanks.

**Review by JoeArmani — APPROVED** [review]
looks good

**Review by anthonypetersen — APPROVED** [review]
looks good

**Pull request merged**